README.md

```diff
@@ -145,23 +158,30 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
     * Exit psql by typing in \q and hitting enter.
 * Migrate
     ```
-
     python manage.py makemigrations caffe_app
     python manage.py migrate
     ```
 8. Install node modules
-```
-npm install
-npm install --save-dev json-loader
-sudo npm install -g webpack
-webpack --progress --watch --colors
-```
-
-9. To start celery worker
-```
-celery -A ide worker --app=ide.celery_app  --loglevel=info
-```
+    ```
+    npm install
+    npm install --save-dev json-loader
+    sudo npm install -g webpack
```

Comment:
```suggestion
    sudo npm install -g webpack@1.15.0
```
Minor suggestion. People usually have issues with webpack version, and we found the above version to work for everyone. Can you add this change?

Documentation looks neat though! Good job.